sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ManagementChannel.java

```diff
@@ -316,10 +348,95 @@ private Message createManagementMessage(String operation, String linkName) {
         return message;
```

Comment:
I should have noticed it before, put the private `renewMessageLock` method into the public one. It is only used in one place. It saves another stack call.

Reply:
I was thinking of when we have to renew multiple lock Token in one go and need to expose that API,  this would be helpful.
But for now I have changed this like you said.